backend/routes/profile.py

```
@@ -0,0 +1,93 @@
+
+router = APIRouter()
+
+# ✅ Predefined fields for validation
+RELEVANT_FIELDS = [
+    "Computer Science", "Electrical Engineering", "Mechanical Engineering",
+    "Civil Engineering", "Artificial Intelligence", "Data Science", "Robotics",
+    "Physics", "Mathematics", "Economics", "Biotechnology"
+]
+
+# ✅ Database Dependency
+def get_db():
```

Ruff (0.8.2)

Missing return type annotation for public function `get_db`

(ANN201)